src/ucp/core/ucp_context.c

```diff
@@ -1543,6 +1641,12 @@ static ucs_status_t ucp_fill_resources(ucp_context_h context,
         ucs_string_set_init(&avail_devices[dev_type]);
     }
 
+    ucs_string_set_init(&aux_tls);
+    status = ucp_fill_aux_tls(&aux_tls);
```

Comment:
maybe better init it later where it is needed? to avoid passing it down thru the stack

Reply:
It is more convenient to have init/cleanup in a single method.
Otherwise, the code will be more complex.

Reply:
@brminich ucp_add_component_resources is called multiple times. Do we want to re-generate aux_tls every time?